requirements.txt

```diff
@@ -1,5 +1,5 @@
 # To pin testing and deployments to specific versions
 fastapi[all]==0.28.0
-lark-parser==0.5.6
+lark-parser==0.7.7
```

Comment:
Ah - it seems that the test issues concerning updating to the latest lark version has indeed been resolved.